### PR TITLE
chore(style): Fix lint errors

### PR DESCRIPTION
**Pull request by erezrokah**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

Fixes remaining lint errors for the new linter.
Should merge after https://github.com/cloudquery/cloudquery/pull/782 and https://github.com/cloudquery/cloudquery/pull/779

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/.github/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run --new-from-rev main` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests 🧪
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: pkg/config/load_config_test.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func TestLoadRemoteFile(t *testing.T) {
 			} else {
 				assert.Equal(t, 1, len(diags))
 				assert.Equal(t, "Failed to read file", diags[0].Summary)
-				assert.Equal(t, fmt.Sprintf("The file \"%s\" could not be read: file does not exist. Hint: Try `cloudquery init <provider>`.", tc.Path), diags[0].Detail)
+				assert.Equal(t, fmt.Sprintf("The file \"%s\" could not be read: file does not exist. Hint: Try `cloudquery init <provider>`", tc.Path), diags[0].Detail)
```
Comment (disq) [inline pkg/config/load_config_test.go:123]: Use `%q` instead of `\"%s\"`

File: pkg/config/config.go [hunk 2]
```diff
@@ -7,59 +7,34 @@ import (
...
 }
 
-func (c Config) GetProvider(name string) (*Provider, error) {
-	for _, p := range c.Providers {
-		if name == p.Alias {
-			return p, nil
-		}
-	}
-	return nil, fmt.Errorf("provider %s does not exist", name)
+type History struct {
```
Comment (disq) [inline pkg/config/config.go:25]: add Deprecated comment here as well

**Comment by erezrokah** [comment]
Converted to draft until I can bring this up to speed

File: pkg/ui/console/diagnostics.go [hunk 3]
```diff
@@ -5,19 +5,19 @@ import (
...
 
+const diagFormat = "%sType: %-10s Severity: %s\n\tSummary: %s\n"
+
 func classifyDiagnostics(dd diag.Diagnostics) diag.Diagnostics {
 	// TODO: placeholder, every diagnostic we see that can be classified to diag.USER should be updated here
 	return dd
 }
 
-func printDiagnostics(header string, dd *diag.Diagnostics, redactDiags, verbose, squash bool) {
+func printDiagnostics(header string, dd *diag.Diagnostics, redactDiags, verbose bool) {
```
Comment (erezrokah) [inline pkg/ui/console/diagnostics.go:20]: In calling methods `squash` is always true

**Review by roneli — APPROVED** [review]
LGTM

**Review by yevgenypats — APPROVED** [review]

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**